src/main/java/com/esri/samples/analysis/line_of_sight_geoelement/LineOfSightGeoElementSample.java

```diff
@@ -0,0 +1,235 @@
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+      stackPane.getChildren().add(sceneView);
+
+      // add base surface for elevation data
+      Surface surface = new Surface();
+      surface.getElevationSources().add(new ArcGISTiledElevationSource("http://elevation3d.arcgis" +
+          ".com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+      scene.setBaseSurface(surface);
+
+      // add buildings from New York City
+      String buildingsURL = "https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/New_York_LoD2_3D_Buildings/SceneServer/layers/0";
+      ArcGISSceneLayer buildings = new ArcGISSceneLayer(buildingsURL);
+      scene.getOperationalLayers().add(buildings);
+
+      // create a graphics overlay for the graphics
+      GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
+      graphicsOverlay.getSceneProperties().setSurfacePlacement(LayerSceneProperties.SurfacePlacement.RELATIVE);
+      sceneView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // set up a heading expression to handle graphic rotation
+      SimpleRenderer renderer3D = new SimpleRenderer();
+      Renderer.SceneProperties renderProperties = renderer3D.getSceneProperties();
+      renderProperties.setHeadingExpression("[HEADING]");
+      graphicsOverlay.setRenderer(renderer3D);
+
+      // create a point graph near the Empire State Building to be the observer
+      Point observationPoint = new Point(-73.9853, 40.7484, 200, SpatialReferences.getWgs84());
+      Graphic observer = new Graphic(observationPoint, new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFFFF0000, 5));
+      graphicsOverlay.getGraphics().add(observer);
+
+      // create a slider to change the observer's Z value
+      Slider heightSlider = new Slider(150, 300, 200);
+      heightSlider.setMaxSize(30, 150);
+      heightSlider.setShowTickLabels(true);
+      heightSlider.setOrientation(Orientation.VERTICAL);
+      stackPane.getChildren().add(heightSlider);
+      StackPane.setAlignment(heightSlider, Pos.TOP_LEFT);
+      StackPane.setMargin(heightSlider, new Insets(10, 0, 0, 10));
+      heightSlider.valueProperty().addListener(e -> {
+        PointBuilder pointBuilder = new PointBuilder((Point) observer.getGeometry());
+        pointBuilder.setZ(heightSlider.getValue());
+        observer.setGeometry(pointBuilder.toGeometry());
+      });
+
+      // create waypoints around a block for the taxi to drive to
+      waypoints = Arrays.asList(
+          new Point(-73.984513, 40.748469, SpatialReferences.getWgs84()),
+          new Point(-73.985068, 40.747786, SpatialReferences.getWgs84()),
+          new Point(-73.983452, 40.747091, SpatialReferences.getWgs84()),
+          new Point(-73.982961, 40.747762, SpatialReferences.getWgs84())
+      );
+
+      // create a graphic of a taxi to be the target
+      String modelURI = new File("./samples-data/dolmus_3ds/dolmus.3ds").getAbsolutePath();
```

Comment:
May need to add this to samples data. I didn't see it appear after refreshing the data.